examples/dielectric.jl

```diff
@@ -19,18 +20,19 @@ model = model_LDA(lattice, atoms, symmetry=:off)
 basis = PlaneWaveBasis(model, Ecut; kgrid=kgrid)
 scfres = self_consistent_field(basis, tol=1e-14)
 
-# Apply ε = 1 - K χ0
-function eps_fun(dv)
-    dv = reshape(dv, size(scfres.ρ.real))
-    dρ = apply_χ0(scfres.ham, scfres.ψ, scfres.εF, scfres.eigenvalues, from_real(basis, dv))
-    Kdρ = apply_kernel(basis, dρ; ρ=scfres.ρ)
-    vec(dv - Kdρ.real)
+# Apply ε = 1 - χ0 (vc + fxc)
```

Comment:
Eps is the dielectric operator, and applies to potentials conventionally. Call that epsT or something?